.github/workflows/pre-release.yml

```diff
@@ -67,6 +71,20 @@ jobs:
         if: github.event.inputs.sdkJavaVersion != ''
         run: ./.tools/run_jvm_tests.sh
 
+      # Bump Python SDK
+      - uses: actions/checkout@v3
+        if: github.event.inputs.sdkPythonVersion != ''
+      - uses: actions/setup-python@v5
+        if: github.event.inputs.sdkPythonVersion != ''
+        with:
+          python-version: "3.12"
+      - name: Bump Python SDK
+        if: github.event.inputs.sdkPythonVersion != ''
+        run: ./.tools/update_python_examples.sh ${{ inputs.sdkPythonVersion }}
```

Comment:
you need to add at the bottom of this file the `requirements.txt` to `add-paths`

Reply:
Hi